src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -186,4 +184,11 @@ export const documentIntegrations: {
       },
     ],
   },
-};
+];
+
+export const documentIntegrations: {
+  [key: string]: DocumentIntegration;
+} = documentIntegrationList.reduce((accum, integration) => {
+  accum[integration.slug] = integration;
+  return accum;
+}, {});
```

Comment:
🚲🏠 --> I personally like `fromEntries()` API, it's more explicit than `reduce()`, about what you're trying to do.

```suggestion
} = Object.fromEntries(documentIntegrationList.map(integration => [integration.slug, integration]));
```